opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap.py

```diff
@@ -25,35 +25,55 @@
 
 # target library to desired instrumentor path/versioned package name
 instrumentations = {
+    "asgi": "opentelemetry-ext-asgi>=0.11b0",
+    "asyncpg": "opentelemetry-ext-asyncpg>=0.11b0",
+    "boto": "opentelemetry-ext-boto>=0.11b0",
+    "botocore": "opentelemetry-ext-botocore>=0.11b0",
+    "celery": "opentelemetry-ext-celery>=0.11b0",
     "dbapi": "opentelemetry-ext-dbapi>=0.8b0",
     "django": "opentelemetry-ext-django>=0.8b0",
+    "elasticsearch": "opentelemetry-ext-elasticsearch>=0.11b0",
     "flask": "opentelemetry-ext-flask>=0.8b0",
```

Comment:
Maybe we can wait until https://github.com/open-telemetry/opentelemetry-python/pull/890 gets merged so we can add fastapi here?